packages/rx_bloc_cli/mason_templates/bricks/lib_mfa/__brick__/lib/l10n/network_lookup/mfa_lookup.dart

```diff
@@ -21,8 +21,6 @@ class AppI18nMfaLookup extends I18nFeatureMfaLookup {
           bundledI18n = I18nFeatureMfa(I18nFeatureMfaLookup_bg());
           break;
         case 'en':
-          bundledI18n = I18nFeatureMfa(I18nFeatureMfaLookup_en());
```

Comment:
Please make sure such changes are addressed in separate PRs. From what I see this is not related to QR code integration, but feel free to correct me if I'm wrong